specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2357 @@
+      "name": "endpoint",
+      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true
+    },
+    "FileStream": {
+      "name": "fileStream",
+      "description": ".json, .pdf, .jpg, .png or .tiff type file stream.",
+      "x-ms-parameter-location": "method",
+      "in": "body",
+      "schema": {
+        "$ref": "#/definitions/SourcePath"
+      }
+    },
+    "PageRange": {
+      "name": "PageRange",
+      "in": "query",
+      "description": "Custom page numbers for multi-page documents(PDF/TIFF), input the number of the pages you want to get OCR result. For a range of pages, use a hyphen. Separate each page or range with a comma or space.",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "(^[0-9]+-[0-9]+$)|(^[0-9]+$)"
+      }
+    },
+    "Locale": {
+      "name": "locale",
+      "in": "query",
+      "description": "Locale of the input document. Supported locales include: en-AU, en-CA, en-GB, en-IN, en-US(default).",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "enum": [
+        "en-AU",
+        "en-CA",
+        "en-GB",
+        "en-IN",
+        "en-US"
+      ],
+      "x-ms-enum": {
+        "name": "Locale",
+        "modelAsString": true
+      }
+    },
+    "Language": {
+      "name": "language",
+      "in": "query",
+      "description": "The BCP-47 language code of the text in the document. Currently, only English ('en'), Dutch (‘nl’), French (‘fr’), German (‘de’), Italian (‘it’), Portuguese (‘pt'), simplified Chinese ('zh-Hans') and Spanish ('es') are supported (print – nine languages and handwritten – English only). Layout supports auto language identification and multi language documents, so only provide a language code if you would like to force the documented to be processed as that specific language.",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "enum": [
+        "en",
+        "es",
+        "de",
+        "fr",
+        "it",
+        "nl",
+        "pt",
+        "zh-Hans",
+        "ja"
+      ],
+      "x-ms-enum": {
+        "name": "Language",
+        "modelAsString": true
+      }
+    },
+    "ReadingOrder": {
+      "name": "readingOrder",
+      "in": "query",
+      "description": "Reading order algorithm to sort the text lines returned. Supported reading orders include: Basic(default), Natural.",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "enum": [
+        "basic",
+        "natural"
+      ],
+      "x-ms-enum": {
+        "name": "ReadingOrder",
+        "modelAsString": true
```

Comment:
I currently do not anticipate having new reading order values in the near future.  Recommend modelAsString=false.

Reply:
done